sydney/sydney.py

```diff
@@ -360,7 +372,11 @@ async def start_conversation(self) -> None:
 
             self.conversation_id = response_dict["conversationId"]
             self.client_id = response_dict["clientId"]
-            self.conversation_signature = response_dict["conversationSignature"]
+            if response_dict.get("conversationSignature"):
```

Comment:
I think this if is not needed anymore, we can keep only the code within the `else`. If it breaks, then it means the API has a breaking change which will likely require more code changes to Sydney.py